cmd/snapd/main.go

```diff
@@ -51,7 +52,12 @@ func init() {
 }
 
 func main() {
-	cmd.ExecInSnapdOrCoreSnap()
+	// In preseed mode re-exec is not used
+	if release.PreseedMode() {
```

Comment:
IMO release is a terrible place for this. It's such a kitchen sink of stuff. I would really prefer a plain function anywhere else, including plain in `cmd/`

Reply:
@zyga As Samuele said, this is not final. If I move it to _cmd_, then right now cmd would need to be imported by managers, which is something we want to avoid I think? And I don't want to introduce a flag passed down to managers because the final approach which will revolve around a "DeviceCtx"-like approach (we will read env close to the startup and set it on state cache, meaning the flag does not need to be passed down to managers).

Reply:
I understand this is temporary. I just wanted to voice my concern.

Note that I explicitly did not -1 this, you can go ahead and merge. I just wanted to not allow this to go on unnoticed as I think this is not the right direction if left unchecked.